### PR TITLE
Change ETH unit to Ether from gwei #24

### PR DESCRIPTION
**Pull request by daiki44**
#24

### CODE REVIEW

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -11,3 +19,23 @@ export function getTokenName(tokenId: number) {
       return 'ETH'
   }
 }
+
+// TODO: create for general
+// TODO: support a decimal
+export function getUnitEther(tokenId: number, amount: BigNumber): number {
```
Comment (tkmct) [inline src/helpers/utils.ts]: I think it's better to use general name.

**Comment by shogochiai** [comment]
Okay this is what I wanted in here.

https://github.com/cryptoeconomicslab/wallet/issues/20#issuecomment-481532636

File: src/components/Heading.tsx [hunk 2]
```diff
@@ -10,23 +10,22 @@ import {
...
   tokenId
 }: {
-  balance: BigNumber
+  balance: number
   wallet: ChamberWallet
   tokenId: number
 }) => {
-  const [balanceInner, setBalanceInner] = React.useState(balance.toNumber())
+  const [balanceInner, setBalanceInner] = React.useState(balance)
   React.useEffect(() => {
```
Comment (shogochiai) [inline src/components/Heading.tsx]: Wow I didn't know this Hooks feature introduced in React v16.8. Good lesson :)

File: src/helpers/utils.ts [hunk 3]
```diff
@@ -11,3 +19,23 @@ export function getTokenName(tokenId: number) {
       return 'ETH'
   }
 }
+
+// TODO: create for general
+// TODO: support a decimal
+export function getTokenMinDigits(tokenId: number, amount: BigNumber): number {
+  switch (tokenId) {
+    case 0:
+      return parseInt(formatEther(parseUnits(amount.toString(), 'gwei')))
```
Comment (shogochiai) [inline src/helpers/utils.ts]: I guess using parseInt will squash important decimal part ;)

Comment (shogochiai) [reply src/helpers/utils.ts]: `parseFloat()` would be good.

Comment (shogochiai) [reply src/helpers/utils.ts]: `commify(amount: Number|String):String` from `ethers/utils` would be suitable if you need rendering-friendly format

File: src/helpers/utils.ts [hunk 4]
```diff
@@ -11,3 +19,23 @@ export function getTokenName(tokenId: number) {
...
+    default:
+      return amount.toNumber()
+  }
+}
+
+export function getTokenMaxDigits(tokenId: number, amount: number): number {
+  switch (tokenId) {
+    case 0:
+      return parseInt(formatUnits(parseEther(amount.toString()), 'gwei'))
```
Comment (shogochiai) [inline src/helpers/utils.ts]: I recommend `parseFloat`

File: src/helpers/utils.ts (hunk 3 above)
Comment (shogochiai) [reply src/helpers/utils.ts]: @daiki44 I'm touching around here. Please be noted.

Comment (shogochiai) [reply src/helpers/utils.ts]: @daiki44 Finished. I made pull-req to this branch. Please merge this before hand.

https://github.com/cryptoeconomicslab/wallet/pull/31#issuecomment-481690368

**Comment by shogochiai** [comment]
closes #24

**Pull request merged**